permissions/plugin-dogu-integration.yml

```diff
@@ -7,3 +7,5 @@ developers:
 - "dogutechio"
 issues:
   - github: *GH
+cd:
+  enabled: true
```

Comment:
An empty line is suggested at the end of this file.

Reply:
Thanks for feedback!